api/src/main/kotlin/com/gmtkgamejam/routing/InfraRoutes.kt

```diff
@@ -0,0 +1,28 @@
+package com.gmtkgamejam.routing
+
+import com.gmtkgamejam.models.posts.PostItem
+import com.gmtkgamejam.respondJSON
+import com.gmtkgamejam.search.OpensearchClusterConfigurer
+import com.gmtkgamejam.services.PostService
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.routing.*
+import org.litote.kmongo.eq
+
+// TODO: Auth control
+fun Application.configureInfraRouting() {
```

Comment:
The `/infra` route is intended to provide simple setup tooling for an administrator to avoid us needing lots of helper scripts or ingrained knowledge on setting things up.

This route will load all posts from the DB and set up the `posts` index in the SE (including special mappings etc). I'd like to keep as much of this setup in code as possible, we'll forget how to do it otherwise.

This routing block will likely use a secret configuration value (set as a runtime environment variable) as access control.